server/app.py

```diff
@@ -74,10 +75,27 @@ def remove_player_from_lobby(data):
     room = data['room']
     player_name = data['playerName']
     # Remove the player from the room
-    ROOMS[room].pop(player_name, None)
```

Comment:
Including `None` as a default value is the preferred way to pop a key from the dictionary. I say we leave this here to avoid any `KeyError` exceptions.